manifests/charts/ai-gateway-helm/templates/serviceaccount.yaml

```diff
@@ -1,4 +1,9 @@
-{{- if .Values.controller.serviceAccount.create -}}
+# Copyright Envoy AI Gateway Authors
+# SPDX-License-Identifier: Apache-2.0
+# The full text of the Apache license is available in the LICENSE file at
+# the root of the repo.
+
+{{ if .Values.controller.serviceAccount.create -}}
```

Comment:
Note that I've removed the leading `-` in the conditional; otherwise the next line was appended directly to the comment instead of having a newline